com.unity.netcode.gameobjects/Tests/Runtime/NetworkShowHideTests.cs

```diff
@@ -134,28 +152,32 @@ public IEnumerator NetworkShowHideTest()
             m_NetSpawnedObject2.Spawn();
             m_NetSpawnedObject3.Spawn();
 
-            yield return new WaitForSeconds(0.1f);
 
-            // get the NetworkObject on a client instance
-            yield return RefreshNetworkObjects();
+            for (int mode = 0; mode < 2; mode++)
```

Comment:
Made this test harder to pass by hiding and showing the objects **both** individually and through a list. Checks a different code path too.